### ISSUE TITLE
Parsing issue

### ISSUE THREAD

**Issue by kirualex** [lokalise/lokalise-fastlane-actions#5]
During the `lokalise_metadata` action I get this error : 
```
sh: -c: line 0: unexpected EOF while looking for matching `"'
sh: -c: line 1: syntax error: unexpected end of file
```
The result is that it seems to cut-off the process at the 2nd language, although it finishes successfully.
On ITC, I have 2 languages out of 4 that are ok.

Here my complete log  : 

```
fastlane update_itunes
[✔] 🚀 
sh: -c: line 0: unexpected EOF while looking for matching `"'
sh: -c: line 1: syntax error: unexpected end of file
/***/fastlane/actions/get_itunes_metadata_from_lokalise.rb:84: warning: key "pt" is duplicated and overwritten on line 85
[18:21:13]: ----------------------------------------
[18:21:13]: --- Step: Verifying fastlane version ---
[18:21:13]: ----------------------------------------
[18:21:13]: Your fastlane version 2.89.0 matches the minimum requirement of 2.18  ✅
[18:21:13]: ------------------------------
[18:21:13]: --- Step: default_platform ---
[18:21:13]: ------------------------------
[18:21:13]: -----------------------
[18:21:13]: --- Step: xcversion ---
[18:21:13]: -----------------------
[18:21:13]: Setting Xcode version to /Applications/Xcode.app for all build steps
[18:21:13]: Driving the lane 'update_itunes' 🚀
[18:21:13]: -------------------------------
[18:21:13]: --- Step: lokalise_metadata ---
[18:21:13]: -------------------------------
{:api_token=>"a***1", :id=>"***", :platform_mask=>16, :keys=>"[\"appstore.app.name\",\"appstore.app.description\",\"appstore.app.keywords\",\"appstore.app.promotional_text\",\"appstore.app.release_notes\",\"appstore.app.subtitle\"]"}
[18:21:14]: Response {JSON} 🚀
sh: -c: line 0: unexpected EOF while looking for matching `"'
sh: -c: line 1: syntax error: unexpected end of file
[18:21:14]: Successfully loaded 'Deliverfile' 📄

+--------------------+--------------------------+
| Detected Values from './fastlane/Deliverfile' |
+--------------------+--------------------------+
| team_id            | ***                |
| skip_screenshots   | true                     |
+--------------------+--------------------------+

[18:21:21]: Login successful

[✔] Activating languages en-US, ru... 
[✔] Uploading metadata to iTunes Connect 
[18:21:48]: Successfully uploaded set of metadata to iTunes Connect

+------+----------------------------+-------------+
|                fastlane summary                 |
+------+----------------------------+-------------+
| Step | Action                     | Time (in s) |
+------+----------------------------+-------------+
| 1    | Verifying fastlane version | 0           |
| 2    | default_platform           | 0           |
| 3    | xcversion                  | 0           |
| 4    | lokalise_metadata          | 39          |
+------+----------------------------+-------------+

[18:21:52]: fastlane.tools finished successfully 🎉
```

**Comment by kirualex** [comment]
I've looked a little deeper and it seems there is an issue with "es" from "es_ES"

**Comment by kirualex** [comment]
Fixed with e6c23c8dea8154e51e3b73a990a53f0ace3f6407
